kollect-typeclasses/src/main/kotlin/kollect/typeclasses/StateT.kt

```diff
@@ -0,0 +1,9 @@
+package kollect.typeclasses
+
+import arrow.Kind
+import arrow.core.Tuple2
+import arrow.data.StateT
+import arrow.typeclasses.Applicative
+
+fun <F, S, A> stateT(AF: Applicative<F>, fa: Kind<F, A>): StateT<F, S, A> =
```

Comment:
This file can be removed because it can be replaced by `StateT.liftF`

Reply:
Done 👍